src/main/java/uk/gov/hmcts/reform/adoption/notification/handlers/ApplicationSubmittedNotificationEventHandler.java

```diff
@@ -24,7 +24,7 @@ public void sendNotificationPostApplicationSubmission(ApplicationSubmitNotificat
         log.info("ApplicationSubmittedNotificationEventHandler triggered");
         sendNotificationService.sendNotifications(
             applicationSubmitNotificationEvent.caseData());
-        log.info("Sent Notifications after citizen application submission for CaseID: {}",
+        log.info("After citizen application submission for CaseID: {}",
                  applicationSubmitNotificationEvent.caseData().getId());
 
```

Comment:
'After' could be any part of the logic after submission.  Could we change to something like:

"Completed attempt at notifications. Citizen application submission for...."